masmini-android/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+plugins {
+    id("com.android.library")
+    kotlin("android")
+}
+
+android {
+    compileSdkVersion(30)
+    buildToolsVersion("29.0.3")
+
+    defaultConfig {
+        minSdkVersion(14)
+        targetSdkVersion(30)
+        versionCode = 1
+        versionName = "1.0"
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+}
+
+dependencies {
+    api(project(":masmini-common"))
+
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1")
+    api("androidx.appcompat:appcompat:1.2.0")
+
+    val lifecycleVersion = "2.3.0-rc01"
+    api("androidx.lifecycle:lifecycle-runtime-ktx:$lifecycleVersion")
+    api("androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleVersion")
+    api("androidx.lifecycle:lifecycle-viewmodel-savedstate:$lifecycleVersion")
+
+
+    testImplementation("junit:junit:4.13.1")
+    androidTestImplementation("androidx.test:runner:1.3.0")
+    androidTestImplementation("androidx.test.espresso:espresso-core:3.3.0")
+}
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile::class.java) {
+    kotlinOptions.freeCompilerArgs += listOf(
+            "-Xuse-experimental=kotlin.Experimental",
+            "-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi",
+            "-Xuse-experimental=kotlinx.coroutines.FlowPreview"
+    )
+}
+
+val sourcesJar by tasks.registering(Jar::class) {
+    @Suppress("UnstableApiUsage")
+    archiveClassifier.set("sources")
+    from(android.sourceSets["main"].java.sourceFiles)
+}
```

Comment:
We may have to extract this to a sub-file so it can be reused for all libs (like we have in Gradle)